doc/source/user_guide/io.rst

```diff
@@ -3449,6 +3449,14 @@ Reading Excel files
 In the most basic use-case, ``read_excel`` takes a path to an Excel
 file, and the ``sheet_name`` indicating which sheet to parse.
 
+* When reading in (``.xlsx``) and (``.xlsm``) files, pandas is using :func:`~openpyxl.load_workbook`.
```

Comment:
The function used is dependent on the engine and not the extension. Even if the relation `extension ~ engine` is well-defined today, it may not be in the future. Can you rephrase as something like:

> For the engine `openpyxl`, pandas is using...

Also, I think it'd be good to mention _why_ we're telling the user this. Something like "When using engine_kwargs, pandas will pass arguments through to the engine. For this, it is important to know which function pandas is using internally."

Reply:
@rhshadrach Would this be acceptable: `For the engine openpyxl, pandas is using :func:`~openpyxl.load_workbook` to read in xlsx and xlsm files.`